pkg/controller/azure.go

```diff
@@ -46,7 +46,7 @@ func (c *Controllers) SetupWithManager(mgr ctrl.Manager) error {
 		&cache.RedisClaimSchedulingController{},
 		&cache.RedisClaimDefaultingController{},
 		&cache.RedisClaimController{},
-		&cache.RedisController{},
+		&cache.Controller{},
```

Comment:
I feel like we should standardize on package names like `controller/cache/redis` at some point so we can have `redis.ClaimSchedulingController{}`, `redis.Controller{}`, etc. Not important for the moment.

Reply:
I'm actually leaned towards having packaged on the claim level. Like `pkg/controller/cache` would have `redis.go`, `memcached.go` managed controllers alongside with one `claim.go` file.

BTW, `RedisController{}` name makes more sense. I remember Goland convinced me that its prefix conflicts with a package name but this seems to be gone now.